libraries/bxdf/translation/standard_surface_to_gltf_pbr.mtlx

```diff
@@ -76,12 +76,18 @@
     </dotproduct>
 
     <!-- Metallic roughness -->
-    <ifequal name="base_color" type="color3">
+    <ifequal name="no_transmission_base_color" type="color3">
```

Comment:
For consistency with other example graphs, I would recommend using camel case names for nodes representing intermediate variables, with snake case used only for the final nodes that match the naming conventions in glTF PBR.

So this variable would be named `noTransmissionBaseColor`, and similarly the new intermediate variables below would be named `transmissionInEffectAttenuationColor` and so forth.